src/Formatter.js

```diff
@@ -39,27 +39,45 @@ function parseFormatStr(formatStr) {
   };
 }
 
+function createCachedFormatter(fn) {
+  const cache = Object.create(null);
```

Comment:
`const cache = {};` is probably faster.

Reply:
no, it is not - `{}` creates object with prototype - when you try to access property it has to go through the whole prototype chain; also if your cache key happens to be one of the Object's properties (like for example `hasOwnProperty`) then it would falsely return cached object (it should not happen here because of key format however)

Reply:
https://www.measurethat.net/Benchmarks/Show/13610/0/object-literal-vs-objectcreatenull 🤷‍♂️